doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -199,6 +199,7 @@ pimcore:
 - [DataObjects][CSV Export] Changed encoding of table data-types to `json_encode` from `base64_encoded`.
 
 ## 10.6.0
+- [DataObject] Added new helper inheritance helper function `DataObject\Serivce::useInheritedValues`
```

Comment:
This PR is assigned to the `10.6` milestone, but targets the `11.x` branch currently.

Reply:
You are right, should be in the Pimcore 11 section/milestone